src/components/application_manager/test/commands/mobile/system_request_test.cc

```diff
@@ -129,7 +150,30 @@ TEST_F(SystemRequestTest, Run_TypeDisallowed_DisallowedResult) {
   command->Run();
 }
 
+TEST_F(SystemRequestTest, Run_FileNameContainSlash_InvalidData) {
+  mobile_apis::RequestType::eType request_type =
+      mobile_apis::RequestType::CLIMATE;
+  (*msg_)[am::strings::msg_params][am::strings::request_type] = request_type;
+  (*msg_)[am::strings::msg_params][am::strings::file_name] = "file_name/";
+  CommandPtr command(CreateCommand<SystemRequest>(msg_));
+  ON_CALL(mock_app_manager_, application(kConnectionKey))
+      .WillByDefault(Return(app_));
+
+  EXPECT_CALL(policy_handler_mock,
+              IsRequestTypeAllowed(kPolicyAppId, request_type))
+      .WillOnce(Return(true));
+  info_ = "Sync file name contains forbidden symbols.";
+  EXPECT_CALL(
+      mock_app_manager_,
+      ManageMobileCommand(
+          MobileResponseIs(mobile_result::INVALID_DATA, info_, false), _));
+
+  command->Run();
+}
+
 TEST_F(SystemRequestTest, Run_FileWithBinaryDataSaveUnsuccessful_GenericError) {
+  mobile_apis::RequestType::eType request_type =
```

Comment:
@okozlovlux in all the other test classes request_type in declared in current test case. I havent seen it declared as const in a namespace. Do you think it's better to move it to a namespace as const? @AByzhynar same question)